fix cuckoo output module

doomedraven opened the pull request.

doomedraven committed:
fix cuckoo output

micheloosterhof reviewed (commented):
Hi! Thanks for submitting this! Seems I missed the import config!

* Could you leave the single quotes in place, please for the config items? I usually use single quotes for short computer string, double quotes for human readable
* Is there a reason you remove the getboolean?
* res="" -> wouldn't res = None be better?
* For return res, wouldn't return res.ok be better?

doomedraven committed:
fixes as required

i leaving res as return so in case of failature it wan't say what res hasn't instance "ok"
